src/modules/auth/LoginForm.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import styled from 'styled-components';
+import Input from '../next-template/components/Input';
```

Comment:
If you create a index for the component, you can shorthand this import to

```ts
import {Input, Button} from '../next-template/components`
```

You can reference `src/common/components/index.ts` for reference.